sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -483,6 +483,41 @@ _nib_offl_entry_t *_nib_offl_iter(const _nib_offl_entry_t *last)
     return NULL;
 }
 
+_nib_offl_entry_t *_nib_pl_add(unsigned iface,
+                               const ipv6_addr_t *pfx,
+                               unsigned pfx_len,
+                               uint32_t valid_ltime,
+                               uint32_t pref_ltime)
+{
+    uint32_t now;
+    _nib_offl_entry_t *dst = _nib_offl_add(NULL, iface, pfx, pfx_len, _PL);
+
+    if (dst == NULL) {
+        return NULL;
+    }
+    assert((valid_ltime >= pref_ltime));
+    if ((valid_ltime != UINT32_MAX) || (pref_ltime != UINT32_MAX)) {
+        now = (xtimer_now_usec64() / US_PER_MS) & UINT32_MAX;
+        if (pref_ltime != UINT32_MAX) {
+            _evtimer_add(dst, GNRC_IPV6_NIB_PFX_TIMEOUT, &dst->pfx_timeout,
+                         pref_ltime);
+            if (((pref_ltime + now) == UINT32_MAX) && (now != 0)) {
+                pref_ltime++;
+            }
+            pref_ltime += now;
+        }
+        if (valid_ltime != UINT32_MAX) {
+            if (((valid_ltime + now) == UINT32_MAX) && (now != 0)) {
```

Comment:
no need to check nor `now != 0`, because it cannot be `0` due to the check above (`valid_ltime != UINT32_MAX`).